doc/getting_started/installation_linux.rst

```diff
@@ -164,23 +176,48 @@ Follow these steps to install the SDK on your Linux host system.
       export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
       export ZEPHYR_SDK_INSTALL_DIR=<sdk installation directory>
 
-  To use the same toolchain in new sessions in the future, you can set the
-  variables in the file :file:`${HOME}/.zephyrrc`, for example:
+   To use the same toolchain in new sessions in the future, you can set the
+   variables in the file :file:`${HOME}/.zephyrrc`, for example:
+
+   .. code-block:: console
+
+      cat <<EOF > ~/.zephyrrc
```

Comment:
I really dislike this here-files or whatever they are called. Can we instead have the contents of the file and its path (~/.zephyrrc) separately so users can choose their editor of choice to create the file?